.github/workflows/publish-release.yml

```diff
@@ -478,6 +482,22 @@ jobs:
           touch .release-env
           make release
 
+      - name: Artifact Attestations
+        id: attestation
+        uses: actions/attest-build-provenance@v1
+        with:
+          subject-path: |
+            dist/zetacored_**/*
+            dist/zetaclientd_**/*
+            dist/checksums.txt
+
+      - name: Upload Attestation Bundle
+        env: 
+          GH_TOKEN: ${{ secrets.PAT_GITHUB_SERVICE_ACCT }}
```

Comment:
Why do we need this token rather than the default token?

Reply:
I was thinking the same thing when working on this. Before there were some restrictions with creating release or something like that but we may not have been setting the permissions correctly. I would like to try going back to the default one. I am going to merge this PR as is and I've added another issue. 

https://github.com/zeta-chain/node/issues/2391

Reply:
Removed custom GH token from this workflow and left an issue to clean up the rest.